### ISSUE TITLE
Wrong series/episode detection

### ISSUE THREAD

**Issue by pannal** [guessit-io/guessit#235]
I've got a user from Sub-Zero (implements subliminal, implements guessit), who'd like to get his episode matched: `Series_name.2005.211.episode.title.avi`.

```
guessit.matchtree: Final result: {
[1.00] "mimetype": "video/x-msvideo",
[1.00] "container": "avi",
[0.24] "title": "211 episode title",
[0.35] "series": "Series_name",
[0.66] "year": 2005,
[1.00] "type": "episode",
[0.30] "season": 2
}
```

Thanks for looking into this!

**Comment by Toilal** [comment]
I'll fix it in guessit 2.x, those kind of naming could find `2005` as year and `211` as episode number.

**Comment by queeup** [comment]
I have same problem. guessit can't guess season and episode from this kind of file naming.
```
ValueError: Insufficient data to process the guess
INFO:subliminal.video:Scanning video 'greys.anatomy.1214.hdtv-lol.mp4' in "tvshows/Grey's Anatomy"
ERROR:subliminal.video:Error scanning video
Traceback (most recent call last):
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 447, in scan_videos
    subtitles_dir=subtitles_dir)
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 301, in scan_video
    video = Video.fromguess(path, guess_file_info(path))
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 104, in fromguess
    return Episode.fromguess(name, guess)
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 174, in fromguess
    raise ValueError('Insufficient data to process the guess')
ValueError: Insufficient data to process the guess
INFO:subliminal.video:Scanning video 'greys.anatomy.1215.hdtv-lol.mp4' in "tvshows/Grey's Anatomy"
ERROR:subliminal.video:Error scanning video
Traceback (most recent call last):
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 447, in scan_videos
    subtitles_dir=subtitles_dir)
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 301, in scan_video
    video = Video.fromguess(path, guess_file_info(path))
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 104, in fromguess
    return Episode.fromguess(name, guess)
  File "/storage/.xbmc/addons/shell.python.subliminal/resources/lib/subliminal/video.py", line 174, in fromguess
    raise ValueError('Insufficient data to process the guess')
ValueError: Insufficient data to process the guess

```

**Comment by Toilal** [comment]
Actualy this particular case (from @queeup) is working with guessit 2.x, which is now merged in [subliminal 2.0-rc1](https://github.com/Diaoul/subliminal/tree/2.0-rc1).

**Comment by ratoaq2** [comment]
Fixed by #446
